usmqe_tests/ui/test_host_ui.py

```diff
@@ -0,0 +1,205 @@
+        LOGGER.debug("Hostname in grafana: {}".format(dashboard_values["host_name"]))
+        LOGGER.debug("Hostname in main UI "
+                     "after dot replacement: '{}'".format(host.hostname.replace(".", "_")))
+        pytest.check(dashboard_values["brick_count"] == host.bricks_count)
+        LOGGER.debug("Brick count in grafana: {}".format(dashboard_values["brick_count"]))
+        LOGGER.debug("Brick count in main UI: {}".format(host.bricks_count))
+        pytest.check(dashboard_values["host_health"] == host.health.lower())
+        LOGGER.debug("Host health in grafana: '{}'".format(dashboard_values["host_health"]))
+        LOGGER.debug("Host health in main UI: '{}'".format(host.health.lower()))
+
+
+def test_brick_dashboard(application, imported_cluster_reuse):
+    """
+    Test Dashboard button of each brick of each host
+    """
+    """
+    :step:
+      Log in to Web UI and get the cluster identified by cluster_member.
+      Get the list of its hosts.
+    :result:
+      Host objects are initiated and their attributes are read from the page
+    """
+    clusters = application.collections.clusters.get_clusters()
+    for cluster in clusters:
+        if cluster.cluster_id == imported_cluster_reuse["cluster_id"]:
+            test_cluster = cluster
+    hosts = test_cluster.hosts.get_hosts()
+    """
+    :step:
+      For each host, get the list of its bricks.
+      For each brick, click its Dashboard button.
+      Check that the correct Grafana dashboard appears
+      and that it shows expected value of brick status.
+    :result:
+      Grafana dashboard is opened, checked for its values and closed.
+      Status check fails due to https://bugzilla.redhat.com/show_bug.cgi?id=1668900
+    """
+    for host in hosts:
+        bricks = host.bricks.get_bricks()
+        for brick in bricks:
+            dashboard_values = brick.get_values_from_dashboard()
+            brick_status = brick.status
+            LOGGER.debug("Cluster name in grafana: {}".format(dashboard_values["cluster_name"]))
+            LOGGER.debug("Cluster name in main UI: {}".format(brick.cluster_name))
+            pytest.check(dashboard_values["cluster_name"] == brick.cluster_name)
+            pytest.check(dashboard_values["host_name"] == brick.hostname.replace(".", "_"))
+            LOGGER.debug("Hostname in grafana: {}".format(dashboard_values["host_name"]))
+            LOGGER.debug("Hostname in main UI "
+                         "after dot replacement: '{}'".format(brick.hostname.replace(".", "_")))
+            pytest.check(dashboard_values["brick_path"] == brick.brick_path.replace("/", ":"))
+            LOGGER.debug("Brick path in grafana: {}".format(dashboard_values["brick_path"]))
+            LOGGER.debug("Brick path in main UI "
+                         "after slash replacement: {}".format(brick.brick_path.replace("/", ":")))
+            pytest.check(dashboard_values["brick_status"] == brick_status)
```

Comment:
Could you add a quick message into the `check()`? This is because in the summary of failure, I see just:

```
usmqe_tests/ui/test_host_ui.py:195: AssumptionFailure
	pytest.check(dashboard_values["host_name"] == brick.hostname.replace(".", "_"))

usmqe_tests/ui/test_host_ui.py:203: AssumptionFailure
	pytest.check(dashboard_values["brick_status"] == brick_status)
```

Details could be found in the full log, as the debug messages provides it, but it should be obvious what the failure is about without digging into the full logs.